packages/traits/index.ts

```diff
@@ -58,6 +58,7 @@ export type StarryUITrait =
  | StarryUITextContentTrait
  | StarryUIThemeTrait
  | StarryUIOnSelectTrait
+ | StarryUICellRenderTrait
```

Comment:
Missing `| StarryUICellDimensionsTrait` here

Reply:
Done

Reply:
❌ @Vikulakav still missing, don't see the change here

Because it is missing, this is causing an issue on Vercel when building the project. https://vercel.com/starryui/starryui/7CPkjLvP23BXP7dDU3PNMg8BdeYm

You can test this locally by running from the root directory: `npm run build && cd packages/starryui-docs && npm run compile`

![Screenshot from 2023-10-29 12-08-49](https://github.com/starryui/starryui/assets/688553/e5561f02-80f4-43f3-8f4f-f786a8c4859c)

Reply:
🟨 I see `StarryUICellDimensionsTrait` is added correctly now but the build is still failing. Needs an investigation.